src/Files.App/Helpers/FilePropertiesHelpers.cs

```diff
@@ -118,5 +121,21 @@ void Navigate(Frame frame)
 				frame.Navigate(typeof(Properties), argument, new SuppressNavigationTransitionInfo());
 			}
 		}
+
+		private static string GetFilesLogoPath()
+		{
+			var appTilesPath = Path.Combine(Package.Current.InstalledLocation.Path, "Assets/AppTiles");
```

Comment:
It could be a good idea to extract the path `Assets/AppTiles` into a constant stored inside the `Constant.cs` since it's a data that may be useful elsewhere.

Reply:
We should be able to link directly to `Assets/AppTiles/Dev/Logo.ico` and the pipeline will edit the path depending on the release config.

Reply:
Other uses of Package.Current.InstalledLocation.Path use this way. It can be modified but it must be done everywhere and in another pr.

Reply:
This is actually the wrong path as the assets are under the packaging project. Regardless, we shouldn't be referencing `Preview` and `Release` as those aren't part of the project. The correct process is to point to `Assets/AppTiles/Dev/Logo.ico`.